examples/leasing/node_release_script.py

```diff
@@ -0,0 +1,187 @@
+#! /bin/python
+"""
+This script will free nodes from a project after
+a specific amount of time.
+
+You can find more information including
+config file and status file format in
+README file.
+"""
+
+import time
+import ConfigParser
+
+from hil.client.client import Client, RequestsHTTPClient
+from hil.client.base import FailedAPICallException
+
+
+class HILClientFailure(Exception):
+    """Exception indicating that the HIL client failed"""
+
+
+class StatusFileError(Exception):
+    """Exception regarding Status File Format"""
+
+
+def hil_client_connect(endpoint_ip, name, pw):
+    """Returns a HIL client object"""
+
+    hil_http_client = RequestsHTTPClient()
+    hil_http_client.auth = (name, pw)
+
+    return Client(endpoint_ip, hil_http_client)
+
+
+def load_node_info(statusfile):
+    """
+    Creates a structure containing all information from the
+    file. We will change and update this data structure and then
+    write it back to the file. In this way, the file will be read
+    and written once.
```

Comment:
1. Describe the data structure.
2. Mention that it's the return value ("creates" doesn't communicate where the data structure ends up; for all the reader knows it could be in some global variable).
3. You probably don't need to mention the bit about reading & writing once; this isn't such a deep thing that you need to talk about alternative designs.